activesupport/lib/active_support/cache.rb

```diff
@@ -184,8 +184,8 @@ def mute
       #
       # If there is no such data in the cache (a cache miss), then +nil+ will be
       # returned. However, if a block has been passed, that block will be passed
-      # the key and executed in the event of a cache miss. The return value of the
-      # block will be written to the cache under the given cache key, and that
+      # to the key and executed in the event of a cache miss. The return value of
```

Comment:
We pass key to the block and not the other way around- https://github.com/rails/rails/blob/c4a46fa781d39f1b4607cb1613a1c67fa044ce54/activesupport/lib/active_support/cache.rb#L293-L293